packages/polaris-viz/src/components/DonutChart/components/InnerValue/InnerValue.tsx

```diff
@@ -47,7 +47,8 @@ export function InnerValue({
     </animated.span>
   );
 
-  const valueToDisplay = activeValue || animatedTotalValue;
+  const activeValueExists = activeValue !== null && activeValue !== undefined;
+  const valueToDisplay = activeValueExists ? activeValue : animatedTotalValue;
```

Comment:
`activeValue` can be `0` and we still want to show that in the center if thats the case.